controllers/selfnoderemediation_controller.go

```diff
@@ -212,27 +212,40 @@ func (r *SelfNodeRemediationReconciler) Reconcile(ctx context.Context, req ctrl.
 	return result, r.updateSnrStatusLastError(snr, err)
 }
 
-func (r *SelfNodeRemediationReconciler) updateSnrProcessingCondition(reason processingChangeReason, snr *v1alpha1.SelfNodeRemediation) error {
-	var conditionStatus metav1.ConditionStatus
-	switch reason {
+func (r *SelfNodeRemediationReconciler) updateSnrTypeConditions(processingTypeReason processingChangeReason, snr *v1alpha1.SelfNodeRemediation) error {
```

Comment:
this also reads strange.... maybe just updateConditions()? (IMHO its clear that it is about SNR CRs, and conditions have a type)